modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -5,6 +5,20 @@
 
 module AppealsApi
   class NoticeOfDisagreement < ApplicationRecord
+    PII_POLICY = lambda {
+      complete_status = COMPLETE_STATUSES.map { |status| "'#{status}'" }.join(', ')
+
+      <<~SQL.squish
```

Comment:
Is this not possible to write in ActiveRecord?

Reply:
I suppose that we could remove all the logic from `AppealsApi::RemovePii#records_to_be_expunged` and delegate everything to the model and that would allow us to use standard AR queries. I wrote this as a way to keep the encrypted data (header and form) queries in `AppealsApi::RemovePii#records_to_be_expunged` and have the appeal specific rules inside the actual models...

Do you think that would be preferable, or did you have something else in mind?

Reply:
I like this structure, but would love to know of other ideas and/or why ActiveRecord might be preferable.

Reply:
I just pushed up a commit that uses AR queries instead of raw sql for us to consider... Is that more of what you had it mind @yakschuss? I like to use AR queries when possible but I'm not sure I prefer it in the NOD's case...